.github/workflows/ci.yml

```diff
@@ -32,6 +32,7 @@ jobs:
       - run-on-ref-slice-fork
       - run-on-tokio-explicit
       - run-on-tokio-implicit
+      - run-on-ref-slice-fork-windows
     steps:
       - run: exit 0
```

Comment:
I tweaked this bit on `main` last night because it was allowing failing CI jobs to auto-merge. You'll need to merge `main` and add the new job in the new conditional around here as well.

Reply:
Ah darn, my PR didn't work. Never mind! Let's merge this (though consider adding the comment I suggested in a separate PR) and I'll deal with editing this job in my PR.